### ISSUE TITLE
Multiple NPCs lvl 66 incorrect HP value

### ISSUE THREAD

**Issue by duzyizly** [Atlantiss/NetherwingBugtracker#9199]
https://www.wowhead.com/tbc/npc=18202/murkblood-putrifier

Lvl 65 has ~4700hp but lvl 66 6,7k 

On this video if You add all damage done - it's less than 5k.
https://youtu.be/iVncTWnQu4U?t=43

**Server Revision**: 
4024

**Comment by duzyizly** [comment]
![WoWScrnShot_121824_110120](https://github.com/user-attachments/assets/54cfd6c5-1aec-46cc-a5fc-4f2aa32509b1)
![WoWScrnShot_121824_110121](https://github.com/user-attachments/assets/a618a8b0-a916-462d-8d14-268123100e42)

https://www.youtube.com/watch?v=tZlOpKhpMxo

same with Nagrand voidspawns lvl 66

**Comment by duzyizly** [comment]
Baically lots of lvl 66 mobs have incorrect HP leveles here in Nagrand, gotta check other places.

![WoWScrnShot_121824_110909](https://github.com/user-attachments/assets/230c01b1-bb00-4119-8d9a-da586cd4b265)
![WoWScrnShot_121824_110912](https://github.com/user-attachments/assets/681895d3-d43b-43ed-9243-4d154cb65038)

**Comment by duzyizly** [comment]
Also lvl 66:
https://www.wowhead.com/tbc/npc=17160/living-cyclone
https://www.wowhead.com/tbc/npc=17155/lake-surger
https://www.wowhead.com/tbc/npc=17137/boulderfist-mage

probably more, better check with database than by roaming Nagrand.

This is also true for npcs from other locations, only lvl 66 mobs.
![WoWScrnShot_121824_223504](https://github.com/user-attachments/assets/e9bbe08b-7577-4cd0-8e1c-ff3fbb658283)
![WoWScrnShot_121824_230948](https://github.com/user-attachments/assets/22cc218a-fb64-46c7-86fe-c9152aebb68b)

**Comment by Godmatik** [comment]
Yes it seems confirmed at least on PTR, lvl 66 mobs have incorrect health scaling
